concepts/Vulnerability_disclosure_and_embargo_policy/00004-Vulnerability_disclosure_and_embargo_policy.md

```diff
@@ -0,0 +1,221 @@
+# RFC00004: Vulnerability Disclosure and Embargo Policy
+- Authors: [axel simon](github@axelsimon.net)
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2020-03-10
+- Status Note: under discussion
+- Supersedes: N/A
+- Start Date: 2020-03-03 (date you started working on this idea)
+- Tags: security, infrastructure
+
+## Summary
+
+Describe the security vulnerability disclosure process, including embargoes.
+
+## Motivation
+
+Users and researchers who discover security issues in Enarx need to know that
+we welcome their efforts and how we will manage their disclosure. They must
+also be able to have an idea of how we use security embargoes.
+
+In other words:
+- We want:
+  - to encourage reporting of security issues (ie: “attack us!”)
```

Comment:
Not entirely happy about "attack us!".  Maybe ("please let us know about vulnerabilities!")

Reply:
I liked the idea of putting the same idea in more abstract terms and in more direct terms, which we have used out loud (hence the quotation marks). However, i agree that there must be a nice way to put it than just "attack us!".